pallets/settlement/src/lib.rs

```diff
@@ -1128,20 +1136,23 @@ impl<T: Config> Module<T> {
                             return TransactionOutcome::Rollback(Ok(Err(*leg_id)));
                         }
                     }
-                    LegAsset::OffChain { .. } => {}
+                    Leg::OffChain { .. } => {}
                 }
             }
         }
         TransactionOutcome::Commit(Ok(Ok(())))
     }
 
+    #[rustfmt::skip]
```

Comment:
NIT: I know we should not add this, but cargo fmt makes the match branches look terrible. I'm ok with removing it though, just let me know.

Reply:
I would prefer that we don't do this.  If `rustfmt` really makes the code difficult to read, we can try to refactor parts into functions.